<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QMainWindow" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="ui6_enemyPokemon">
    <property name="geometry">
     <rect>
      <x>521</x>
      <y>240</y>
      <width>211</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string>敌方宝可梦贴图</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ui6_HP_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>50</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="ui6_ATK">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>80</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="ui6_SPD">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>140</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="ui6_ATK_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>80</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="ui6_DEF">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>110</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="ui6_DEF_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>110</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="ui6_SPD_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>140</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="ui6_HP">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>50</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
   </widget>
   <widget class="QLabel" name="ui6_myPokemon">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>211</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string>我方宝可梦贴图</string>
    </property>
   </widget>
   <widget class="QToolButton" name="ui4_returnMainUi">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QToolButton" name="ui5_return">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QToolButton" name="ui3_returnMainUi">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QToolButton" name="ui5_levelUp">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>250</width>
      <height>380</height>
     </rect>
    </property>
    <property name="text">
     <string>
升级赛


选择宝可梦与服务器进行虚拟决斗

可以为宝可梦获得经验

胜利不会获得宝可梦

失败也不会失去宝可梦</string>
    </property>
   </widget>
   <widget class="QToolButton" name="ui5_battle">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>250</width>
      <height>380</height>
     </rect>
    </property>
    <property name="text">
     <string>
决斗赛


选择宝可梦与服务器进行虚拟决斗

可以为宝可梦获得经验

胜利可以获得该战胜的宝可梦

失败则由用户选择失去一个宝可梦</string>
    </property>
   </widget>
   <widget class="QLabel" name="ui5_hint">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>hint</string>
    </property>
   </widget>
   <widget class="QToolButton" name="ui6_normalAttack">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>450</y>
      <width>65</width>
      <height>85</height>
     </rect>
    </property>
    <property name="text">
     <string>普攻</string>
    </property>
   </widget>
   <widget class="QToolButton" name="ui6_skill">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>450</y>
      <width>65</width>
      <height>85</height>
     </rect>
    </property>
    <property name="text">
     <string>技能</string>
    </property>
   </widget>
   <widget class="QLabel" name="ui6_turn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="ui6_bullet">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>290</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="ui7_winLose">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>351</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>winLose</string>
    </property>
   </widget>
   <widget class="QLabel" name="ui7_pokemon">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>251</width>
      <height>251</height>
     </rect>
    </property>
    <property name="text">
     <string>pokemon</string>
    </property>
   </widget>
   <widget class="QLabel" name="ui7_prize">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>150</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>prize</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ui7_ack">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>420</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QLabel" name="ui7_exp">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>320</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>exp</string>
    </property>
   </widget>
   <widget class="QLabel" name="ui6_bullet_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>290</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="ui6_damage">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>160</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>damage</string>
    </property>
   </widget>
   <widget class="QLabel" name="ui6_mark">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>190</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>mark</string>
    </property>
   </widget>
   <widget class="QToolButton" name="ui6_enemyTurn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>360</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="ui6_returnMainUi">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QToolButton" name="ui2_myPokemon">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>120</width>
      <height>140</height>
     </rect>
    </property>
    <property name="text">
     <string>宝可梦</string>
    </property>
   </widget>
   <widget class="QToolButton" name="ui2_prize">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>180</y>
      <width>120</width>
      <height>140</height>
     </rect>
    </property>
    <property name="text">
     <string>战绩</string>
    </property>
   </widget>
   <widget class="QToolButton" name="ui2_saveAndQuit">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>400</y>
      <width>120</width>
      <height>140</height>
     </rect>
    </property>
    <property name="text">
     <string>保存并退出</string>
    </property>
   </widget>
   <widget class="QToolButton" name="ui2_battle">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>200</y>
      <width>210</width>
      <height>230</height>
     </rect>
    </property>
    <property name="text">
     <string>战斗</string>
    </property>
   </widget>
   <widget class="QToolButton" name="ui2_allPlayers">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>120</width>
      <height>140</height>
     </rect>
    </property>
    <property name="text">
     <string>玩家</string>
    </property>
   </widget>
   <widget class="QToolButton" name="ui2_returnSignIn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>400</y>
      <width>120</width>
      <height>140</height>
     </rect>
    </property>
    <property name="text">
     <string>登出</string>
    </property>
   </widget>
   <widget class="QLabel" name="ui8_silver2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>330</y>
      <width>121</width>
      <height>161</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>拥有10只满级宝可梦时获得</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="ui8_bronze1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>121</width>
      <height>161</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>拥有5只宝可梦时获得</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QToolButton" name="ui8_win">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>120</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>胜场：</string>
    </property>
   </widget>
   <widget class="QLabel" name="ui8_silver1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>121</width>
      <height>161</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>拥有10只宝可梦时获得</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="ui8_golden1">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>121</width>
      <height>161</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>拥有15只宝可梦时获得</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QToolButton" name="ui8_lose">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>270</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>负场：</string>
    </property>
   </widget>
   <widget class="QToolButton" name="ui8_per">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>420</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>胜率：</string>
    </property>
   </widget>
   <widget class="QLabel" name="ui8_bronze2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>121</width>
      <height>161</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>拥有5只满级宝可梦时获得</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="ui8_golden2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>330</y>
      <width>121</width>
      <height>161</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>拥有15只满级宝可梦时获得</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QToolButton" name="ui8_returnMainUi">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QLabel" name="hint">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>hint</string>
    </property>
   </widget>
   <widget class="QToolButton" name="ui6_autobattle">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>340</y>
      <width>70</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>自动战斗</string>
    </property>
   </widget>
   <widget class="QLabel" name="ui6_enemyfire">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>480</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="ui6_myfire">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>480</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="ui8_normal">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>260</y>
      <width>201</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>宝可梦徽章</string>
    </property>
   </widget>
   <widget class="QLabel" name="ui8_super">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>500</y>
      <width>201</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>高级宝可梦徽章</string>
    </property>
   </widget>
   <widget class="QToolButton" name="ui5_rules">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>50</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="ui6_rules">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>40</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="ui1_background">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>245</y>
      <width>220</width>
      <height>230</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ui1_signUp">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>410</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>注册并登录</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ui1_userName">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>281</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ui1_label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>280</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>用户名：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ui1_signIn">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>370</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ui1_password">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>320</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="ui1_label_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>320</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>密码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="ui1_img4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>370</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="ui1_img1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="ui1_img3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>210</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="ui1_trademark">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>445</width>
      <height>236</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="ui1_img2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>370</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
